app/script/view_model/VideoCallingViewModel.js

```diff
@@ -184,12 +145,12 @@ z.viewModel.VideoCallingViewModel = class VideoCallingViewModel {
         if (!isVisibleId) {
           this.visibleCallId = callEntity.id;
 
-          if (this.showLocalVideo() || this.showRemoteVideo()) {
+          if (this.showRemoteVideo()) {
             this.multitasking.isMinimized(false);
             return this.logger.info(`Maximizing video call '${callEntity.id}' to full-screen`, callEntity);
           }
 
-          this.multitasking.isMinimized(true);
+          //this.multitasking.isMinimized(true);
```

Comment:
@AndyLnd arf, I believe we have to handle that case. I couldn't really understand what this was about but that condition broke (it minimized the call every time).

Maybe we don't need this anymore. What do you think?